README.md

```diff
@@ -153,6 +181,7 @@ To translate between LLVM IR and SPIR-V:
     * `-o file_name` - to specify output name
     * `-spirv-debug` - output debugging information
     * `-spirv-text` - read/write SPIR-V in an internal textual format for debugging purpose. The textual format is not defined by SPIR-V spec.
+    * `--spirv-tools-dis` - print SPIR-V assembly in standard format. Only available on [builds with SPIR-V](#build-with-spirv-tools).
```

Comment:
I don't think there is a really _standard_ format for textual representation of SPIR-V.

Also a typo in the link
```suggestion
    * `--spirv-tools-dis` - print SPIR-V assembly in standard format. Only available on [builds with SPIRV-Tools](#build-with-spirv-tools).
```

Reply:
Hmm, although I thought that SPIRV-Tools produce the standard syntax. I am using this terminology in clang docs too. I think it will be more confusing if we start talking about SPIRV-Tools syntax because then it is not clear why would anyone bother about some syntax from another project. We should bear in mind that this documentation is not for top experts in the tooling aspects of this area.

Reply:
ok, I had the impression that SPIRV-Tools produced a defacto standard syntax because it is used in the examples in the spec... but if we don't align on this then I would say we should not write overstatements in the public docs.

Let's go for` SPIRV-Tools syntax` for now even though I don't like exposing the implementation details to the users... but at least it is not an incorrect statement.